rekallpy/rekall/predicates.py

```diff
@@ -0,0 +1,966 @@
+"""
```

Comment:
This is a rather long file. It would be helpful to have a list of exported predicates in the module docstring.

Reply:
It shows up fine in the documentation - I can't figure out how to get a list of functions in the module in the sidebar like in the pyspark documentation, but that would be the best. Otherwise the docstring would get stale with new functionality.